pyproject.toml

```diff
@@ -1,6 +1,11 @@
 [build-system]
-build-backend = "flit_core.buildapi"
-requires = ["flit_core >=3.4,<4"]
+build-backend = "hatchling.build"
```

Comment:
I'm fine with switching but out of curiosity, why the switch?

Reply:
With the change to absolute imports flit couldn't find the `__version__` string as it requires importing the package. `hatchling` allows specification of the file that contains the version string, so it doesn't have the same problem.